### DISCUSSION TITLE
Help please - Am unable to login to my supabase account

### DISCUSSION THREAD

**Discussion by rbkayz** [supabase/supabase#7356] [Questions] [answered]
Every time I click on sign-in, it redirects me to the login page

https://app.supabase.com/#

What can I do to fix this? Ty

**Comment by Inzeppelin** [comment, marked as answer]
Confirm, we can't login to supabase account either. API also doesn't work due to cors error. Looks like it's global issue and probably it's related to cloudflare https://www.cloudflarestatus.com/

**Reply by rbkayz** [reply]
Yep. Thanks. Learnt that this exists as well - https://status.supabase.com/
